web/react/components/search_results_item.jsx

```diff
@@ -19,6 +19,11 @@ export default class SearchResultsItem extends React.Component {
         e.preventDefault();
 
         EventHelpers.emitPostFocusEvent(this.props.post.id);
+
+        if ($(window).width() < 768) {
```

Comment:
Hey guys I've added this so that on mobile devices the RHS gets closed and the center channel can be seen by users when Jump is clicked from search results.